src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml.cpp

```diff
@@ -18,21 +21,105 @@ using namespace Windows::UI::Xaml::Input;
+    { NumbersAndOperatorsEnum::InvCsc, { L"arccsc()", 7, 0 } },
+    { NumbersAndOperatorsEnum::InvCot, { L"arccot()", 7, 0 } },
+    { NumbersAndOperatorsEnum::Sinh, { L"sinh()", 5, 0 } },
+    { NumbersAndOperatorsEnum::Cosh, { L"cosh()", 5, 0 } },
+    { NumbersAndOperatorsEnum::Tanh, { L"tanh()", 5, 0 } },
+    { NumbersAndOperatorsEnum::Sech, { L"sech()", 5, 0 } },
+    { NumbersAndOperatorsEnum::Csch, { L"csch()", 5, 0 } },
+    { NumbersAndOperatorsEnum::Coth, { L"coth()", 5, 0 } },
+    { NumbersAndOperatorsEnum::InvSinh, { L"arcsinh()", 8, 0 } },
+    { NumbersAndOperatorsEnum::InvCosh, { L"arccosh()", 8, 0 } },
+    { NumbersAndOperatorsEnum::InvTanh, { L"arctanh()", 8, 0 } },
+    { NumbersAndOperatorsEnum::InvSech, { L"arcsech()", 8, 0 } },
+    { NumbersAndOperatorsEnum::InvCsch, { L"arccsch()", 8, 0 } },
+    { NumbersAndOperatorsEnum::InvCoth, { L"arccoth()", 8, 0 } },
+    { NumbersAndOperatorsEnum::Abs, { L"abs()", 4, 0 } },
+    { NumbersAndOperatorsEnum::Floor, { L"floor()", 6, 0 } },
+    { NumbersAndOperatorsEnum::Ceil, { L"ceiling()", 8, 0 } },
+    { NumbersAndOperatorsEnum::Pi, { L"\u03C0", 1, 0 } },
+    { NumbersAndOperatorsEnum::Euler, { L"e", 1, 0 } },
+    { NumbersAndOperatorsEnum::XPower2, { L"^2", 2, 0 } },
+    { NumbersAndOperatorsEnum::Cube, { L"^3", 2, 0 } },
+    { NumbersAndOperatorsEnum::XPowerY, { L"^", 1, 0 } },
+    { NumbersAndOperatorsEnum::TenPowerX, { L"10^", 3, 0 } },
+    { NumbersAndOperatorsEnum::LogBase10, { L"log()", 4, 0 } },
+    { NumbersAndOperatorsEnum::LogBaseE, { L"ln()", 3, 0 } },
+    { NumbersAndOperatorsEnum::Sqrt, { L"sqrt()", 5, 0 } },
+    { NumbersAndOperatorsEnum::CubeRoot, { L"cbrt()", 5, 0 } },
+    { NumbersAndOperatorsEnum::YRootX, { L"root(x,n)", 7, 1 } },
+    { NumbersAndOperatorsEnum::TwoPowerX, { L"2^", 2, 0 } },
+    { NumbersAndOperatorsEnum::LogBaseX, { L"log(b, x)", 4, 1 } },
+    { NumbersAndOperatorsEnum::EPowerX, { L"e^", 4, 0 } },
+    { NumbersAndOperatorsEnum::Abs, { L"abs()", 4, 0 } },
+    { NumbersAndOperatorsEnum::X, { L"x", 1, 0 } },
+    { NumbersAndOperatorsEnum::Y, { L"y", 1, 0 } },
+    { NumbersAndOperatorsEnum::OpenParenthesis, { L"(", 1, 0 } },
+    { NumbersAndOperatorsEnum::CloseParenthesis, { L")", 1, 0 } },
+    { NumbersAndOperatorsEnum::Equals, { L"=", 1, 0 } },
+    { NumbersAndOperatorsEnum::Divide, { L"/", 1, 0 } },
+    { NumbersAndOperatorsEnum::Multiply, { L"*", 1, 0 } },
+    { NumbersAndOperatorsEnum::Subtract, { L"-", 1, 0 } },
+    { NumbersAndOperatorsEnum::Add, { L"+", 1, 0 } },
+    { NumbersAndOperatorsEnum::Invert, { L"1/", 2, 0 } },
+    { NumbersAndOperatorsEnum::Negate, { L"-", 1, 0 } },
+    { NumbersAndOperatorsEnum::GreaterThan, { L">", 1, 0 } },
+    { NumbersAndOperatorsEnum::GreaterThanOrEqualTo, { L"\u2265", 1, 0 } },
+    { NumbersAndOperatorsEnum::LessThan, { L"<", 1, 0 } },
+    { NumbersAndOperatorsEnum::LessThanOrEqualTo, { L"\u2264", 1, 0 } },
+    { NumbersAndOperatorsEnum::Zero, { L"0", 1, 0 } },
```

Comment:
Should we also be calling localizationSettings.GetDigitSymbolFromEnUsDigit for each of these? Also will richedit understand numbers from other languages? See #322.

Reply:
I think rich edit will, but I'm not sure the graph will.  I'm currently leaning towards doing this as a followup.